cohttp-eio/src/client.ml

```diff
@@ -44,7 +44,8 @@ let write_request request writer body =
   Buf_write.string writer "\r\n";
   Rwer.write_headers writer headers;
   Buf_write.string writer "\r\n";
-  Body.write_body ~write_chunked_trailers:true writer body
+  Body.write_body ~write_chunked_trailers:true writer body;
+  Buf_write.flush writer
```

Comment:
Can you add this in a separate commit, and mention on the the fix in the changelog?

Reply:
Done. Moved this bit to a separate PR (https://github.com/mirage/ocaml-cohttp/pull/949) - which needs to be merged before this one.